drivers/impedance-analyzer/ad5933/ad5933.h

```diff
@@ -107,6 +115,7 @@
 /* AD5933 Specifications */
 #define AD5933_INTERNAL_SYS_CLK     16000000ul      // 16MHz
 #define AD5933_MAX_INC_NUM          511             // Maximum increment number
+#define AD5933_MAX_SETTLING_CYCLES  511
```

Comment:
Is this new define really needed given that it has the same value as `AD5933_MAX_INC_NUM` ?